### ISSUE TITLE
Support to HIPAA and NIST 800 53 rule groups

### ISSUE THREAD

**Issue by crd1985** [wazuh/wazuh#3326]
This issue is related to https://github.com/wazuh/wazuh-ruleset/issues/392 and https://github.com/wazuh/wazuh-ruleset/issues/394. 

Some changes in `analysisd` are needed to add HIPAA and NIST 800 53 as independent rule groups, similar to PCI DSS or GDPR.

### Update

A related task is needed to add the compliance fields to the JSON alerts related to SCA checks. SCA policies are mapped with several compliance controls (PCI DSS, CIS, etc.) but controls related to each SCA check is included in dynamic fields as `sca.check.compliance`. An example:

We have to extract fields located at `data.sca.check.compliance` for including them into the rule compliance groups.

For cases such as `pci_dss` where can appear in both sides, they should be merged, for unknown compliance controls coming from the SCA policies, they should be added as new arrays in the rule information.

Let see an example of how it should work:

### Before the change

```
{
  "timestamp": "2019-05-29T03:58:30.79-0700",
  "rule": {
    "level": 3,
    "description": "System audit for web-related vulnerabilities: Web vulnerability - .htaccess file compromised - auto append",
    "id": "19009",
    "firedtimes": 27,
    "mail": false,
    "groups": [
      "sca"
    ],
    "gdpr": [
      "IV_35.7.d"
    ],
    "pci_dss": [
      "10.2.7",
      "10.6.1"
     ]
  },
  "agent": {
    "id": "000",
    "name": "ubuntu"
  },
  "manager": {
    "name": "ubuntu"
  },
  "id": "1559127510.104531",
  "decoder": {
    "name": "sca"
  },
  "data": {
    "sca": {
      "type": "check",
      "scan_id": "462559780",
      "policy": "System audit for web-related vulnerabilities",
      "check": {
        "id": "1015",
        "title": "Web vulnerability - .htaccess file compromised - auto append",
        "compliance": {
          "pci_dss": "6.5, 6.6, 11.4",
          "cis_csc": "9.2",
          "invented_control": "ABC.123"
        },
        "references": "https://blog.sucuri.net/2011/05/understanding-htaccess-attacks-part-1.html",
        "directory": [
          "/var/www",
          "/var/htdocs",
          "/home/httpd",
          "/usr/local/apache",
          "/usr/local/apache2",
          "/usr/local/www"
        ],
        "status": "Not applicable",
        "reason": "Could not open '/var/htdocs': No such file or directory"
      }
    }
  },
  "location": "sca"
}
```

### After the change

```
{
  "timestamp": "2019-05-29T03:58:30.79-0700",
  "rule": {
    "level": 3,
    "description": "System audit for web-related vulnerabilities: Web vulnerability - .htaccess file compromised - auto append",
    "id": "19009",
    "firedtimes": 27,
    "mail": false,
    "groups": [
      "sca"
    ],
    "gdpr": [
      "IV_35.7.d"
    ],
    "pci_dss": [
      "10.2.7",
      "10.6.1",
      "6.5",
      "6.6",
      "11.4"
     ],
     "cis_csc": [
       "9.2"
     ],
     "invented_control": [
       "ABC.123"
     ]
  },
  "agent": {
    "id": "000",
    "name": "ubuntu"
  },
  "manager": {
    "name": "ubuntu"
  },
  "id": "1559127510.104531",
  "decoder": {
    "name": "sca"
  },
  "data": {
    "sca": {
      "type": "check",
      "scan_id": "462559780",
      "policy": "System audit for web-related vulnerabilities",
      "check": {
        "id": "1015",
        "title": "Web vulnerability - .htaccess file compromised - auto append",
        "compliance": {
          "pci_dss": "6.5, 6.6, 11.4",
          "cis_csc": "9.2",
          "invented_control": "ABC.123"
        },
        "references": "https://blog.sucuri.net/2011/05/understanding-htaccess-attacks-part-1.html",
        "directory": [
          "/var/www",
          "/var/htdocs",
          "/home/httpd",
          "/usr/local/apache",
          "/usr/local/apache2",
          "/usr/local/www"
        ],
        "status": "Not applicable",
        "reason": "Could not open '/var/htdocs': No such file or directory"
      }
    }
  },
  "location": "sca"
}
```

**Comment by Skeptor** [comment]
I'm working in the branch https://github.com/wazuh/wazuh/tree/update-3326 for the updated tasks.

The commit https://github.com/wazuh/wazuh/commit/b404a0f419f0ba6ba4151db3c2b0833aadabfedf adds the SCA compliance groups to the rule groups, merging the information of the coincident groups and creating new sections for new groups.

![image](https://user-images.githubusercontent.com/9034923/58628350-bb18f580-82d9-11e9-9c8b-91ba5c8477d6.png)

**Comment by chemamartinez** [comment]
Merged #3427
